readme.md

```diff
@@ -8,7 +8,7 @@ Thanks to everyone who continues to contribute; new languages are created every
 Make sure to see [contributing.md](/contributing.md) for instructions on contributing to the project!
 
 <!--Languages start-->
-## Languages (909 total)
+## Languages (911 total)
```

Comment:
```suggestion
## Languages (910 total)
```

Reply:
No, 911 is correct

Reply:
You should run script instead of changing by hand to avoid mistake.